django_datadog_logger/formatters/datadog.py

```diff
@@ -47,10 +68,6 @@ def determine_version(request):
     if hasattr(request, "version"):
         if request.version is not None:
             return str(request.version)
-    elif hasattr(request, "accepted_types"):
-        for t in request.accepted_types:
-            if t.params.get("version") is not None:
-                return unicode_http_header(t.params.get("version"))
```

Comment:
this part of code is required as fallback for getting the version from accept headers, it cannot be removed